test/ProgressOnderwijsUtils.Drawing.Test/ProgressOnderwijsUtils.Drawing.Test.csproj

```diff
@@ -4,6 +4,7 @@
     <DebugSymbols>True</DebugSymbols>
     <OutputType>library</OutputType>
     <TargetFrameworks>net461</TargetFrameworks>
+    <IsPackable>false</IsPackable>

```

Comment:
wie weet is dit op de een of andere manier te onderdrukken via de directorystructuur?  `test/`?

Reply:
(Als dat ueberhaupt beter is...)

Reply:
dotnet pack accepteert geen patterns lijkt het. Het lijkt er ook op dat de test-projecten sowieso niet gepackt worden (de exe's wel). De test-templates in Visual Studio bevatten dan wel weer <IsPackable>false</IsPackable>.